packages/node/src/indexer/fetch.service.ts

```diff
@@ -215,10 +250,15 @@ export class FetchService implements OnApplicationShutdown {
   }
 
   updateDictionary(): void {
-    this.dictionaryQueryEntries = this.getDictionaryQueryEntries();
+    this.buildDictionaryEntriesMap();
+    let checkDictionaryQueryEntries: boolean;
+    this.mappedDictionaryQueryEntries.forEach((value) => {
+      if (value?.length) {
```

Comment:
Im not sure this works. We could have dictionary disabled at a later time if a block filter exists